packages/manager/apps/pci-load-balancer/src/api/hook/pools.hook.ts

```diff
@@ -0,0 +1,54 @@
+import { useMutation } from '@tanstack/react-query';
```

Comment:
i see the team already named hook files such as "usePools" etc so i think it should be better to stick with just one naming convention